wger/settings_global.py

```diff
@@ -88,6 +87,8 @@
 
     # django-bower for installing bower packages
     'djangobower',
+
```

Comment:
unnecessary space, remove it or add a comment indicating what social_django is for